docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -19,10 +19,8 @@ We will start with a single cluster and move to multiple clusters to illustrate
 * Setup a `./controller-config.env` file in the root of the repo with the following key values
 
     ```bash
-    # this sets up your default managed zone
-    AWS_DNS_PUBLIC_ZONE_ID=<AWS ZONE ID>
-    # this is the domain at the root of your zone (foo.example.com)
-    ZONE_ROOT_DOMAIN=test.hcpapps.net # replace this with your root domain
+    AWS_DNS_PUBLIC_ZONE_ID=<AWS ZONE ID> # this sets up your default managed zone
+    ZONE_ROOT_DOMAIN=test.hcpapps.net # this is the domain at the root of your zone (foo.example.com)
```

Comment:
@jasonmadigan i applied the blindly copying logic to this and it still results in a malformed domain in the managedzone cr after running `make local-setup` before doing any further steps. is the idea to address it by adding validation as suggested in https://github.com/Kuadrant/multicluster-gateway-controller/issues/323 or would it be better to copy the text that we've got in the README here https://github.com/Kuadrant/multicluster-gateway-controller#1-running-the-controller-in-the-cluster that might be a bit more bullet proof for noobs.

Reply:
I had to wipe a bunch of resources and restart the controller for a non-malformed config to be picked up

Reply:
I didn't get to the point of starting the controller. Steps I took were
- copy and paste into the file 
- run make local-setup
- kubectl get managedzone -n multi-cluster-gateways -o yaml
- and the spec.domainname included the commented out section.

Reply:
cool. i'll continue with the steps. 👍

Reply:
when I get to `(export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)` I get the following error `export: not valid in this context: (foo.example.com)`

Reply:
It sort of sounds like your `controller-config.env` is still broken - can you try it without any included comments?

Reply:
after fixing the file to have the correct values i'm facing this error `{"error": "InvalidInput: 1 validation error detected: Value 'Z0860675CQXHTYO3UOX3 # this sets up your default managed zone' at 'resourceId' failed to satisfy constraint: Member must have length less than or equal to 32\n\tstatus code: 400, request id: 708873d1-7ee5-424e-b1df-4906b6099964"}`

Reply:
re: `"InvalidInput: 1 validation error detected: Value 'BLAHBLAH # this sets up your default managed zone'` where is BLAHBLAH coming from? sounds like it could be from either aws-credentials.env or controller-config.env

Reply:
yeah. BLAHBLAH is just the key value but I changed it here. in the error message it's a proper ID. I think changing it make sense. i really applied the blindly copying mindset so that a newbie wouldn't get caught out. 👍